files/__addonLocation__/package.json

```diff
@@ -25,6 +25,9 @@
     "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
     "prepack": "rollup --config"
   },
+  "peerDependencies": {
```

Comment:
highly likely a v2 addon will want a peer on ember-source

Reply:
Seems good to me, especially since the v1 blueprint also has this, but I remember [this comment](https://github.com/embroider-build/addon-blueprint/pull/101#issuecomment-1410776746) by @ef4 🤔 

> the main reason people are forced to add it is if they want to say dependencySatisfies('ember-source', ...).

but this is the case quite often, often implicitly by others doing this, e.g. see https://github.com/ember-polyfills/ember-cached-decorator-polyfill/issues/212

So I'm :+1:, but just wanted to make sure...

Reply:
hmmm that's a good point. I'll hopefully remember to bring this up in tomorrows meeting.